packages/components/src/button/button.styles.ts

```diff
@@ -569,7 +569,6 @@ export const buttonStyles: (
     appearanceBehavior(
       'accent',
       css`
-        :host([appearance='accent'][disabled]),
```

Comment:
Is there a specific reason for changing the accent type case?

Reply:
For consistency.
And maybe we should also remove the `background-color` property of `:host([disabled])` too.

Reply:
I would keep it for now as in the opposite of the _stealth_ appearance, the background-color is controlled by the toolkit.